src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,154 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import {
+  ArrowRight,
+  CreditCard,
+  Lock,
+  MoreHorizontal,
+  Tool,
+  Truck,
+  ShoppingBag,
+} from 'react-feather';
+import CartItem from './CartItem/CartItem';
 import './Cart.scss';
 
-function Cart() {
-  return <div>Cart.js</div>;
-}
+const Cart = () => {
+  const [cartData, setCartData] = useState([]);
+  useEffect(() => {
+    fetch('http://localhost:3000/data/cartData.json', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    })
+      .then(response => response.json())
+      .then(result => {
+        setCartData(result);
+      });
+  }, []);
+
+  const totalCost = cartData
+    .map(item => item.cost * item.count)
+    .reduce((acc, cur) => acc + cur, 0);
+
+  const handleAmountChange = (id, num) => {
+    const updatedData = cartData.map(item =>
+      id === item.id ? { ...item, count: item.count + num } : item
+    );
+    setCartData(updatedData);
+  };
+
+  const handleDelete = id => {
+    const updatedData = cartData.filter(item => item.id !== id);
+    window.confirm('정말 삭제하시겠습니까?') && setCartData(updatedData);
+  };
+
+  const handleSubmit = () => {
+    console.log(cartData);
+  };
+
+  return (
+    <div className="cart">
+      <div className="container">
+        <div className="left">
+          <div className="content-left">
+            <div className="page-header">
+              <h1 className="title">
+                장바구니{!cartData.length && '가 비어있어요'}
+              </h1>
+              <MoreHorizontal />
+            </div>
+            {cartData.length && (
+              <>
+                <div className="sub">주문을 어떻게 받고 싶으세요?</div>
+                <div className="choice">
+                  <div className="option">
+                    <Truck /> &nbsp;&nbsp; 배송
+                  </div>
+                  <div className="option">
+                    <ShoppingBag />
+                    &nbsp;&nbsp; 픽업
+                  </div>
+                </div>
+              </>
+            )}
+            {cartData &&
+              cartData.map(cartItem => (
+                <CartItem
+                  id={cartItem.id}
+                  key={cartItem.id}
+                  name={cartItem.name}
+                  image={cartItem.image}
+                  cost={cartItem.cost}
+                  description={cartItem.description}
+                  dimension={cartItem.dimension}
+                  count={cartItem.count}
```

Comment:
```suggestion
                  key={cartItem.id}
                  {...cartItem}
```

Props로 cartItem에 있는 모든 값을 넘겨줄 때 key와 value의 값이 동일하다면
이와 같은 방법으로 한번에 넘겨줄 수 있습니다!